docker-compose-sync.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
```

Comment:
This whole file is  an overlay, which `docker-sync` puts "on top of" our normal `docker-compose.yml`. So `docker-sync-stack start` is essentially

```
docker-compose -f docker-compose.yml -f docker-compose-sync.yml up
```